src/components/sidenav/demoBasicUsage/index.html

```diff
@@ -4,11 +4,10 @@
   <section layout="row" flex>
 
     <md-sidenav
-        class="md-sidenav-left"
+        class="md-sidenav-left md-whiteframe-4dp"
```

Comment:
@DevVersion updated only to fix text hiding. Both forms of `md-whiteframe` API usage (attribute and class) preserved.